### PR TITLE
#45 Stream window wrong size on Macs with multiple displays and different DPIs

### PR DESCRIPTION
**Pull request by psseelman**
- Open stream window full screen regardless of display scale
- Auto resizes stream views

ezgif-7-d0c9a2e3ff4b

### CODE REVIEW

**Comment by chromakode** [comment]
Thanks for taking a look at this! When testing on Windows, it looks like the overlay is not resizing along with the views. Are you able to reproduce this on macOS?

image

I think that making the window resizable is a great change, but sizing the window to full screen is probably not the right default, because Streamwall is oriented towards streamers, who will want a 1080p source to capture. When I filed #45, it was in response to reports that the Streamwall window was opening the wrong size (due to scaling factors making 1 "pixel" > 1px, I think). There's a workaround used in https://github.com/steveseguin/electroncapture/blob/5e7d1cdacecbaf84d06744ff2795b38a8d68d80a/main.js#L9 which I'd like to try, though it's been difficult for me to reproduce this issue. To reproduce, I think you have to set a scaling factor in the display settings and then try to capture using OBS. I've heard reports that the image captured by OBS is a different size from 1080p.

**Comment by psseelman** [comment]
0c43dff changes:
- Remove resizing changes
- Add a scaling step to index.js to take whatever window.width and window.height arguments (defaulted to 1920x1080) and apply display scaling if necessary. Tested with OBS to ensure appropriate sizing

Screen Shot 2020-08-12 at 4 45 16 PM

File: src/node/index.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import TOML from '@iarna/toml'
 import * as Y from 'yjs'
 import { create as createJSONDiffPatch } from 'jsondiffpatch'
 import { Repeater } from '@repeaterjs/repeater'
-import { app, shell, session, BrowserWindow } from 'electron'
+import { app, shell, session, BrowserWindow, screen } from 'electron'
```
Comment (chromakode) [inline src/node/index.js:7]: Nit: could you please remove this change? It looks to be no longer required.

**Comment by chromakode** [comment]
Merged in https://github.com/chromakode/streamwall/commit/83621cec9e5250b289763cfddc069116ec93e308

**Pull request closed without merging**